zulipterminal/model.py

```diff
@@ -164,12 +164,11 @@ def __init__(self, controller: Any) -> None:
         # Register to the queue before initializing further so that we don't
         # lose any updates while messages are being fetched.
         self._fetch_initial_data()
-
```

Comment:
This is an unrelated change. It's worth checking on GitHub before/after you submit/update a PR, or locally on your computer using eg. `tig` or other tools, for unrelated changes in a given commit/PR.